### PR TITLE
Additional links

### PR DESCRIPTION
**Pull request by relu91**
Just minor links to Zion and WAM video.

### CODE REVIEW

**Comment by egekorkan** [comment]
@relu91 I just added a line break to the video to align with others. Otherwise, looks good to me :)

**Review by egekorkan — APPROVED** [review]

**Comment by egekorkan** [comment]
For others, the video thumbnail is not shown due to the fact that the relative URLs do not work with netlify. It works on local renders which is what github is using

File: docs/videos/index.html [hunk 1]
```diff
@@ -62,6 +62,16 @@ <h4>Short Introduction to Web of Things</a></h4>
 		</a>
         </div>
     </div>
+	<br /><br />
+    <div class="row">
+        <div class="col-md-6">
+		<a target="_blank" href="https://www.youtube.com/watch?v=bPxIfZo7jns">
+			<img class="img-responsive" style="min-width: 180px; width: 100%; border-width: thin; border-style: solid;" src="{{'/images/video-thumbnail-wam.png' | absolute_url }}" alt="WAM Video" />
+			<h4>Building a WoT application: WoT application Manager and TD-code</h4>
+			<!-- 2020-9-18 -->
```
Comment (danielpeintner) [inline docs/videos/index.html:71]: If the date is correct it should come first...

Comment (egekorkan) [reply docs/videos/index.html:71]: yes I agree we should be fair and put more recent videos on top.

**Pull request merged**